psi4/src/export_mints.cc

```diff
@@ -1231,7 +1231,8 @@ void export_mints(py::module& m) {
         .def("save_string_xyz_file", &Molecule::save_string_xyz_file, "Saves an XYZ file to arg2")
         .def("save_string_xyz", &Molecule::save_string_xyz, "Saves the string of an XYZ file to arg2")
         .def("Z", &Molecule::Z, py::return_value_policy::copy,
-             "Nuclear charge of atom arg0 (0-indexed without dummies)")
+             "Nuclear charge of atom arg0 (0-indexed without dummies), by default ghost atoms return zero nuclear charge",
+             py::arg("atom"), py::arg("zero_ghost") = true)
```

Comment:
I see that `py::arg("atom")` has been removed upstream; is this intentional? If so I can also remove it. @loriab